include/linux/slaunch.h

```diff
@@ -592,17 +576,32 @@ static inline void slaunch_finalize(int do_sexit)
 	(void)do_sexit;
 }
 
+#endif /* !IS_ENABLED(CONFIG_SECURE_LAUNCH) */
```

Comment:
Same as above, why following functions were moved out of this config option?

Reply:
This is interesting, I'm not sure how it doesn't fail, given that non-inline functions shouldn't be built: https://github.com/TrenchBoot/linux/blob/6f00b5b7d95a2908823936939ce9bf93fd109e72/arch/x86/kernel/Makefile#L77. Am I missing something obvious here?